displayLinac.py

```diff
@@ -71,17 +71,22 @@ def createFaults(self):
             elif level == "CM":
                 prefix = self.cryomodule.pvPrefix
 
+            elif level == "ALL":
+                prefix = csvFaultDict["PV Prefix"]
+
             else:
                 raise (SpreadsheetError("Unexpected fault level in fault spreadsheet"))
 
             tlc = csvFaultDict["Three Letter Code"]
             okCondition = csvFaultDict["OK If Equal To"]
             faultCondition = csvFaultDict["Faulted If Equal To"]
+            suffix = csvFaultDict["PV Suffix"]
```

Comment:
Was this somehow not here before...?

Reply:
We didn't need it until this point! Nearly all the faults had unique TLCs, and those that did share a TLC could be differentiated by their rack and fault/ok condition.